earth_enterprise/rpms/build.gradle

```diff
@@ -55,7 +55,9 @@ def openGeeVersion =
     findAll { !it.isEmpty() }. // Skip empty lines.
     findResult {
         it // The first string that remains is the version string.
-    }
+    }.
+    replaceAll(/-.*release/, '').
```

Comment:
Have you checked to make sure that stripping this out will still allow a "release" RPM to upgrade an alpha or beta RPM? The original reason we included "release" was because of the belief that without it RPM (cause it's dumb) will not consider it a higher version.